### ISSUE TITLE
Dimension out of range (expected to be in range of [-1, 0], but got 1)

### ISSUE THREAD

**Issue by ioritree** [kijai/ComfyUI-WanVideoWrapper#645]
scheduler select "flowmatch_causvid" ,got this problem.

`
# ComfyUI Error Report
## Error Details
- **Node ID:** 27
- **Node Type:** WanVideoSampler
- **Exception Type:** IndexError
- **Exception Message:** Dimension out of range (expected to be in range of [-1, 0], but got 1)
## Stack Trace
```
  File "G:\AI\ComfyUI_windows_portable\ComfyUI\execution.py", line 349, in execute
    output_data, output_ui, has_subgraph = get_output_data(obj, input_data_all, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "G:\AI\ComfyUI_windows_portable\ComfyUI\execution.py", line 224, in get_output_data
    return_values = _map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True, execution_block_cb=execution_block_cb, pre_execute_cb=pre_execute_cb)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "G:\AI\ComfyUI_windows_portable\ComfyUI\execution.py", line 196, in _map_node_over_list
    process_inputs(input_dict, i)

  File "G:\AI\ComfyUI_windows_portable\ComfyUI\execution.py", line 185, in process_inputs
    results.append(getattr(obj, func)(**inputs))
                   ^^^^^^^^^^^^^^^^^^^^^^^^^^^^

  File "G:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-WanVideoWrapper\nodes.py", line 3467, in process
    temp_x0 = sample_scheduler.step(
              ^^^^^^^^^^^^^^^^^^^^^^

  File "G:\AI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-WanVideoWrapper\wanvideo\utils\basic_flowmatch.py", line 50, in step
    (self.timesteps.unsqueeze(0) - timestep.unsqueeze(1)).abs(), dim=1)
                                   ^^^^^^^^^^^^^^^^^^^^^

```
## System Information
- **ComfyUI Version:** 0.3.40
- **Arguments:** ComfyUI\main.py --use-sage-attention --windows-standalone-build
- **OS:** nt
- **Python Version:** 3.12.7 (tags/v3.12.7:0b05ead, Oct  1 2024, 03:06:41) [MSC v.1941 64 bit (AMD64)]
- **Embedded Python:** true
- **PyTorch Version:** 2.7.0+cu128
## Devices

- **Name:** cuda:0 NVIDIA GeForce RTX 3090 : native
  - **Type:** cuda
  - **VRAM Total:** 25769148416
  - **VRAM Free:** 13354480128
  - **Torch VRAM Total:** 16768827392
  - **Torch VRAM Free:** 6329020928`

**Comment by Vanlams** [comment]
Same issue after updating.

**Comment by kijai** [comment]
Should work again now.
